tests/e2e/redis/basic-ingest-redis-serving.py

```diff
@@ -1390,6 +1390,50 @@ def is_different_jobs():
         time.sleep(1)
 
 
+@pytest.mark.run(order=80)
+def test_basic_register_entity_success(client):
```

Comment:
I'm not sure we need entity tests in e2e and especially redis-serving e2e

Reply:
Removed.